tests/rptest/utils/rpenv.py

```diff
@@ -21,6 +21,12 @@ def sample_license():
     if license is None:
         is_ci = os.environ.get("CI", "false")
         assert is_ci == "false"
+        if assert_exists:
+            assert license, (
```

Comment:
nit: `assert !assert_exists or license` ?

Reply:
`assert not assert_exists or license`  🐍

Reply:
Done. We actually know that license is `None` here, so it can be even simpler: `assert not assert_exists`

Reply:
`not` `!=` `!` 🐍